pkg/grub/rootfs.cfg

```diff
@@ -266,6 +266,13 @@ function set_arm64_baremetal {
          set_global dom0_console "console=ttymxc0,115200 earlycon=ec_imx6q,0x30860000,115200"
       fi
    fi
+   if [ "$smb_vendor" = "phytec" ]; then
```

Comment:
Maybe it make sense to provide root config strategies for different vendors? i.e. we will have pkg/grub/rootfs/phytec_rootfs.cfg , ... etc? If we will have a lot of vendors to support it might make sense, because branching in that case will get way worse :)

Reply:
It can be improved, but not in the scope of this PR (which introduces a lot of other stuffs). I would prefer to make it in a different PR and let this change as it is because it's following the current approach.

Reply:
I've created an [issue](https://github.com/lf-edge/eve/issues/3026) just not to forget

Reply:
> I've created an [issue](https://github.com/lf-edge/eve/issues/3026) just not to forget

Perfect @uncleDecart , thanks.